tox.ini

```diff
@@ -121,20 +121,18 @@ commands =
 
 [testenv:typechecks]
 basepython = python3
-skip_install = True
 deps =
-    mypy
-    mypy-zope
+    mypy==1.3.0
+    # When 0.9.2 comes out it will work with 1.3, it's just unreleased at the moment...
+    git+https://github.com/shoobx/mypy-zope
```

Comment:
We could pin a revision here so it doesn't break when they commit something that breaks compatibility with mypy 1.3.0.